accounts/abi/abi.go

```diff
@@ -97,7 +97,6 @@ func (abi *ABI) UnmarshalJSON(data []byte) error {
 		Type      string
 		Name      string
 		Constant  bool
-		Indexed   bool
```

Comment:
I don't understand why you remove this from the list of fields to be un-marshaled? I understand  that it is derived from another source, but what if the field is still present in some client code?

Reply:
Indexed only makes sense in the context of an event field (which is inside "inputs"). In the outer ABI context there's no notion of "indexed" afaik.